command/resource/read/read.go

```diff
@@ -86,10 +86,6 @@ func (c *cmd) Run(args []string) int {
 			return 1
 		}
 	} else {
-		if len(args) < 2 {
```

Comment:
Why was this removed? 🤔

Reply:
The length check was wrapped inside of the function call
https://github.com/hashicorp/consul/blob/b85c9e77bed00d371b2274f8c0ac36e9184a4ef6/command/resource/read/read.go#L90C1-L90C1

https://github.com/hashicorp/consul/blob/b85c9e77bed00d371b2274f8c0ac36e9184a4ef6/command/resource/helper.go#L154

Reply:
Ahh.. I see. Thank you for cleaning that up.